ui/inbox.lua

```diff
@@ -1,10 +1,11 @@
 -- translation
 local S = minetest.get_translator("mail")
 
-
 function mail.show_inbox(name, sortfieldindex, sortdirection, filter)
-    sortfieldindex = tonumber(sortfieldindex or mail.selected_idxs.sortfield[name]) or 3
-    sortdirection = sortdirection or mail.selected_idxs.sortdirection[name] or "1"
+    sortfieldindex = tonumber(sortfieldindex or mail.selected_idxs.sortfield[name])
+    or mail.get_setting(name, "defaultsortfield") or 3
+    sortdirection = sortdirection or mail.selected_idxs.sortdirection[name]
+    or tostring(mail.get_setting(name, "defaultsortdirection")) or "1"
```

Comment:
This last part does not make sense because of `tostring`, fallback value is never used.

Reply:
So I'll explain, `tostring` converts **any** value to string. Also `nil` or `false` or `{}`.
So what do you think this would return?
```lua
local thing = tostring(nil) or "default"
```